From e4b8dd8350ae928cad07ac31fd4844ef794db543 Mon Sep 17 00:00:00 2001
From: Gene Czarcinski <gczarcinski@gmail.com>
Date: Thu, 31 Jan 2013 23:58:53 +0330
Subject: [PATCH 07/18] Add option for less logging (#893997)

Syslog receives many debug messages from os-prober. Added environment
variable OS_PROBER_DISABLE_DEBUG allowing them to be turned off.

Resolves: rhbz#893997
---
 common.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/common.sh b/common.sh
index b9b3951..19ef90a 100644
--- a/common.sh
+++ b/common.sh
@@ -76,7 +76,9 @@ warn() {
 }
 
 debug() {
-  log "debug: $@"
+  if [ -z "$OS_PROBER_DISABLE_DEBUG" ]; then
+    log "debug: $@" 
+  fi
 }
 
 result () {
-- 
2.5.5

